network-libp2p/Cargo.toml

```diff
@@ -51,8 +51,9 @@ nimiq-utils = { workspace = true, features = [
 ] }
 nimiq-validator-network = { workspace = true }
 
+# For now we need to use a fork
 [target.'cfg(not(target_family = "wasm"))'.dependencies]
-libp2p = { git = "https://github.com/jsdanielh/rust-libp2p.git", default-features = false, features = [
+libp2p = { git = "https://github.com/jsdanielh/rust-libp2p.git", branch = "libp2p_0.53.1", default-features = false, features = [
```

Comment:
It'd probably be nicer to stay on master. Cargo.lock will ensure that you don't accidentally change commits.

Reply:
You mean this fork master? It is because that fork's master has changes that are no longer necessary and are not part of libp2p in any way so I can't rebase it

Reply:
I see.